protos/escrow/escrow.proto

```diff
@@ -0,0 +1,110 @@
+syntax = "proto3";
+
+package escrow;
+
+// golang option
+option go_package = "escrowpb";
+// java options
+option java_multiple_files = true;
+option java_outer_classname = "EscrowProto";
+option java_package = "io.btfs.escrow";
+
+import "google/protobuf/timestamp.proto";
+
+service EscrowService {
+  rpc SubmitContracts(EscrowContractRequest) returns (EscrowContractResult);
+  rpc PayIn(PayinRquest) returns (PayinResult);
+  rpc IsPaid(ContractID) returns (PayinStatus);
+  rpc CancelContracts(CancelContractRequest) returns (CancelContractResult);
+}
+
+enum TokenType {
+  BTT = 0;
+}
+
+enum Schedule {
+  MONTHLY = 0;
+  QUARTERLY = 1;
+  ANNUALLY = 2;
+}
+
+message ContractID {
+  bytes contract_id = 1;
+  bytes address = 2;
+  bytes signature = 3;
+}
+
+message EscrowContract {
+  bytes contract_id = 1; // identity of a contract
```

Comment:
What if host want to ask for contract, not asking for paid, but checking or verifying when or how much it would get paid, what rpc call should use?

Reply:
host has the contract, host can know the payment schedule and contract start time to know when it get paid. You are asking query the payout info. Shall we add one API for that? @taiyangc

Reply:
Added GetPayOutStatus API and PayoutStatus define.

Reply:
lgtm, @ShellyWEI could you check if your concerns are resolved?